reboot-initial-setup-gui.sh

```diff
@@ -15,12 +15,15 @@
 # License and may only be used or replicated with the express permission of
 # Red Hat, Inc.
 #
-TESTTYPE="reboot initial-setup gh527"
+TESTTYPE="reboot initial-setup"
 
 . ${KSTESTDIR}/functions.sh
 
 additional_runner_args() {
+    # Wait for reboot and shutdown of the VM,
+    # but exit after the specified timeout.
     echo "--wait"
+    get_timeout()
```

Comment:
I think you need to remove the brackets.
For me this would see more readable (though I am not sure how clean bash would it be)
`echo "--wait $(get_timeout)"`

Reply:
Fixed.